src/js/components/pager/pager.ts

```diff
@@ -45,25 +49,25 @@ export class PagerBehavior extends Behavior {
         return Number(pagerEl.dataset.currentPage) === Number(pagerEl.dataset.totalPages);
     }
 
-    addPage(pagerEl, displayPageStr, disable: boolean ) {
+    addPage(listEl, pagerEl, displayPageStr, disable: boolean ) {
         const li = document.createElement('li');
         if (!disable) {
-            li.appendChild(this.createLink(displayPageStr, pagerEl));
+            if (displayPageStr == Number(pagerEl.dataset.currentPage) ) {
+                li.setAttribute("aria-current", "page")
+            }
+            li.appendChild(this.createLink(displayPageStr));
         } else {
             li.appendChild(this.createSpan(displayPageStr));
         }
-        pagerEl.appendChild(li);
+        listEl.appendChild(li);
     }
 
-    createLink(displayPageStr, pagerEl) {
+    createLink(displayPageStr) {
         const link = document.createElement('a');
         link.appendChild(document.createTextNode(displayPageStr));
         link.setAttribute("href", "javascript:setPage(" + displayPageStr + ");");
         link.setAttribute("aria-label", this.setAriaLabel(displayPageStr));
```

Comment:
Adding the aria-label does not hurt anything in this context, but it is not really necessary for the Next/Previous links.